setup.py

```diff
@@ -58,6 +58,7 @@ def get_packages(package):
         'Jinja2>=2.10.1',
         'livereload>=2.5.1',
         'lunr[languages]==0.5.6',  # must match lunr.js version included in search
+        'nltk<=3.4.5' # must be lower than 3.5.0 which breaks install. See issue 2062 on Github
```

Comment:
Does this actually work? I'm not sure if order is observed here but I do know that `nltk` should be installed first. Then, when lunr is installed it will see that nltk is already installed and not try to install it. To be on the safe side, I would list it before lunr, not after.

Reply:
Also, why not `<3.5.0` which would accept any new bug fix releases in the `3.4` series?

Reply:
On the first question:
I have to confess that I don't know python enough to correctly answer the question. I noticed the deps were ordered alphabetically so I just went with that.

I did however check the build output in the checks and it looked fined.
I would assume the order in here does not matter, python prolly just resolves all of the requirements regardless of the order.

Question 2:
Right! Good remark, will fix that.